### DISCUSSION TITLE
Add support for multiple accounts at different servers

### DISCUSSION THREAD

**Discussion by yahooooza** [astubenbord/paperless-mobile#78] [Feature Requests]
Paperless -ngx doesnt have a good way to seperate documents between users. In that case it can be usefull to set up multiple instances of paperless to do that. A way to quickly switch between accounts would be a nice to have. Thanks in advance :D.

**Comment by astubenbord** [comment]
Thanks for the idea, I was also thinking about something similar. Just to be clear, you simply want a mechanism so that a single user can switch between different accounts on different servers, right? So the app just has to store different login information (server address, username, password, and optionally client certs) which a single user can quickly switch between?

**Comment by yahooooza** [comment]
Yes exactly.

**Comment by astubenbord** [comment]
I think given the progress in https://github.com/paperless-ngx/paperless-ngx/discussions/295 I would prefer to wait before I implement sth. that is essentially just a workaround for an even more encompassing feature. Switching between servers would not really stand in the way of a multi user support feature, however, I just think that many users have it set up this way (multiple instances for multiple users) because it is currently not natively supported.

**Comment by steinm** [comment]
2.0.1 seems to have some initial support for multiple accounts. At least it has a link to add another account. Is this supposed to work already? There is no reaction at all, when I touch on it.

**Reply by astubenbord** [reply]
No, not yet. It's just a placeholder for. the functionality of the upcoming multi-user-permissions update.

**Reply by astubenbord** [reply]
@steinm keep an eye on #148 ;-)

**Comment by steinm** [comment]
Am Thu, Apr 20, 2023 at 11:36:29PM +0000 schrieb Anton Stubenbord:
> @steinm keep an eye on #148 ;-)
I will :-)

-- 
  MMK GmbH, Fleyer Str. 196, 58097 Hagen
  ***@***.***
  Tel: 02331 840446    Fax: 02331 843920

**Comment by astubenbord** [comment]
Implemented in 2.2.0
